src/components/Camera/Permissions/Primer/style.css

```diff
@@ -1,4 +1,4 @@
-@import (less) "../../Theme/constants.css";
```

Comment:
how does webpack generate the css selector with components in nested folders? @stefaniacardenas maybe you know?

Reply:
I think it will be just `onfido-sdk-ui-PermissionsPrimer-classname`

Reply:
okay, so we are running the risk of style collision by using nested folders. There might be a workaround on webpack where we can use the full path, rather than just the folder name.

Reply:
Yes, good point.